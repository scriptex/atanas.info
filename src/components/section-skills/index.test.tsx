import * as React from 'react';
import { render } from '@testing-library/react';

import { SectionSkills } from '.';
import { mockUseEffect } from '../../scripts/shared';

describe('SectionSkills component', () => {
	mockUseEffect();

	it('Should render the SectionSkills component', () => {
		const { asFragment } = render(<SectionSkills />);

		expect(asFragment()).toMatchSnapshot();
	});
});
