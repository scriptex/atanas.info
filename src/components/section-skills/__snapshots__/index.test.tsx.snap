// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionSkills component Should render the SectionSkills component 1`] = `
<Component
  actions={
    <Unknown
      onClick={[Function]}
    >
      Static
       mode
    </Unknown>
  }
  hasButton={true}
  hasShell={true}
  id="skills"
>
  <Component
    name="skills"
  />
  <h2>
    Skills 
    <br />
    <small
      hidden={false}
    >
      (Drag the bubbles to play)
    </small>
  </h2>
  <div
    className=""
    id="skills-graph"
  />
  <div
    className="o-grid is--hidden"
  >
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="0"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-agile"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Agile
          </strong>
          Since 
          2010
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="1"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-angular"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Angular
          </strong>
          Since 
          2017
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="2"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={32}
            name="svg-angularjs"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Angular JS
          </strong>
          Since 
          2014
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="3"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={18}
            name="svg-aws"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Amazon Web Services
          </strong>
          Since 
          2019
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="4"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#222",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={12}
            name="svg-babel"
            width={31}
          />
        </figure>
        <p>
          <strong>
            Babel
          </strong>
          Since 
          2015
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="5"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-babylonjs"
            width={30}
          />
        </figure>
        <p>
          <strong>
            BabylonJS
          </strong>
          Since 
          2018
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="6"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#bdc9ec",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-backbone"
            width={30}
          />
        </figure>
        <p>
          <strong>
            BackboneJS
          </strong>
          Since 
          2013
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="7"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#48992a",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-bash"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Bash
          </strong>
          Since 
          2015
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="8"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#000",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={24}
            name="svg-bem"
            width={29}
          />
        </figure>
        <p>
          <strong>
            BEM
          </strong>
          Since 
          2013
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="9"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#ccc",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-bootstrap"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Bootstrap
          </strong>
          Since 
          2012
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="10"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#122858",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-css"
            width={30}
          />
        </figure>
        <p>
          <strong>
            CSS3
          </strong>
          Since 
          2010
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="11"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#7d7d7d",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-chartjs"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Chart JS
          </strong>
          Since 
          2014
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="12"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#ddd",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-circleci"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Circle CI
          </strong>
          Since 
          2019
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="13"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#97daa8",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={29}
            name="svg-cordova"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Cordova
          </strong>
          Since 
          2014
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="14"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-confluence"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Confluence
          </strong>
          Since 
          2017
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="15"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#5a4126",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-d3"
            width={30}
          />
        </figure>
        <p>
          <strong>
            D3
          </strong>
          Since 
          2012
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="16"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-deno"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Deno
          </strong>
          Since 
          2020
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="17"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={27}
            name="svg-eslint"
            width={30}
          />
        </figure>
        <p>
          <strong>
            ESLint
          </strong>
          Since 
          2016
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="18"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={25}
            name="svg-expressjs"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Express JS
          </strong>
          Since 
          2017
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="19"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-firebase"
            width={25}
          />
        </figure>
        <p>
          <strong>
            Firebase
          </strong>
          Since 
          2018
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="20"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#141e1f",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={32}
            name="svg-foundation"
            width={22}
          />
        </figure>
        <p>
          <strong>
            Foundation
          </strong>
          Since 
          2103
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="21"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#663399",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-gatsby"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Gatsby
          </strong>
          Since 
          2019
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="22"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#f9d7d7",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-git"
            width={30}
          />
        </figure>
        <p>
          <strong>
            GIT
          </strong>
          Since 
          2011
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="23"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={29}
            name="svg-github"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Github Actions
          </strong>
          Since 
          2020
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="24"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-gitlab-logo"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Gitlab CI/CD
          </strong>
          Since 
          2017
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="25"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#f39341",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-grunt"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Grunt
          </strong>
          Since 
          2013
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="26"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#07267b",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={40}
            name="svg-gulp"
            width={40}
          />
        </figure>
        <p>
          <strong>
            Gulp
          </strong>
          Since 
          2013
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="27"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#52260c",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-html"
            width={30}
          />
        </figure>
        <p>
          <strong>
            HTML5
          </strong>
          Since 
          2010
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="28"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#3a3a4f",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={29}
            name="svg-highcharts"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Highcharts
          </strong>
          Since 
          2016
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="29"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-ionic"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Ionic
          </strong>
          Since 
          2014
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="30"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#f0db4f",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-javascript"
            width={30}
          />
        </figure>
        <p>
          <strong>
            JavaScript
          </strong>
          Since 
          2011
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="31"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-jest"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Jest
          </strong>
          Since 
          2017
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="32"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-jira"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Jira
          </strong>
          Since 
          2017
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="33"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#ccc",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-json"
            width={28}
          />
        </figure>
        <p>
          <strong>
            JSON
          </strong>
          Since 
          2013
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="34"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#7bdffd",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-jquery"
            width={30}
          />
        </figure>
        <p>
          <strong>
            jQuery
          </strong>
          Since 
          2011
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="35"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#b6ecec",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-less"
            width={30}
          />
        </figure>
        <p>
          <strong>
            LESS
          </strong>
          Since 
          2012
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="36"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={20}
            name="svg-markdown"
            width={32}
          />
        </figure>
        <p>
          <strong>
            Markdown
          </strong>
          Since 
          2014
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="37"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#3fa",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={26}
            name="svg-mjml"
            width={30}
          />
        </figure>
        <p>
          <strong>
            MJML
          </strong>
          Since 
          2019
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="38"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#353134",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={24}
            name="svg-azure"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Microsoft Azure
          </strong>
          Since 
          2019
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="39"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#1d4002",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-nodejs"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Node JS
          </strong>
          Since 
          2016
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="40"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#7d7d7d",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-parcel"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Parcel Bundler
          </strong>
          Since 
          2019
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="41"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#f6c9fb",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-php"
            width={30}
          />
        </figure>
        <p>
          <strong>
            PHP
          </strong>
          Since 
          2013
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="42"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#333",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-photoshop"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Photoshop
          </strong>
          Since 
          2010
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="43"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#b0f1dc",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-postcss"
            width={30}
          />
        </figure>
        <p>
          <strong>
            PostCSS
          </strong>
          Since 
          2014
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="44"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-react"
            width={30}
          />
        </figure>
        <p>
          <strong>
            React
          </strong>
          Since 
          2016
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="45"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#222",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-redux"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Redux
          </strong>
          Since 
          2016
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="46"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#9c3b2d",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={19}
            name="svg-redux-saga"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Redux Saga
          </strong>
          Since 
          2018
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="47"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#f7f7f7",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-rollup"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Rollup
          </strong>
          Since 
          2018
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="48"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#faf4cf",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-sass"
            width={30}
          />
        </figure>
        <p>
          <strong>
            SASS
          </strong>
          Since 
          2012
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="49"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#ffb13b",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-svg"
            width={30}
          />
        </figure>
        <p>
          <strong>
            SVG
          </strong>
          Since 
          2011
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="50"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#2b1306",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={22}
            name="svg-svn"
            width={30}
          />
        </figure>
        <p>
          <strong>
            SVN
          </strong>
          Since 
          2010
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="51"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#291212",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-sketch"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Sketch
          </strong>
          Since 
          2017
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="52"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#4d4d4e",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-sublime-text"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Sublime Text
          </strong>
          Since 
          2010
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="53"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#dee",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-travisci"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Travis CI
          </strong>
          Since 
          2017
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="54"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#007ACC",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-typescript"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Typescript
          </strong>
          Since 
          2017
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="55"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#000",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={26}
            name="svg-vercel"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Vercel
          </strong>
          Since 
          2019
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="56"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#fff",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-vscode"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Visual Studio Code
          </strong>
          Since 
          2017
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="57"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#4a5c71",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={26}
            name="svg-vue"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Vue
          </strong>
          Since 
          2016
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="58"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#0d2748",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-webpack"
            width={30}
          />
        </figure>
        <p>
          <strong>
            Webpack
          </strong>
          Since 
          2015
        </p>
      </div>
    </div>
    <div
      className="o-grid__item xs-12 sm-6 md-4 lg-3"
      key="59"
    >
      <div
        className="c-skill"
      >
        <figure
          style={
            Object {
              "backgroundColor": "#111",
            }
          }
        >
          <Component
            className="c-skill__icon"
            height={30}
            name="svg-wordpress"
            width={30}
          />
        </figure>
        <p>
          <strong>
            WordPress
          </strong>
          Since 
          2013
        </p>
      </div>
    </div>
  </div>
</Component>
`;
